gensim/sklearn_api/__init__.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+"""Scikit learn wrapper for gensim.
+Contains various gensim based implementations which match with scikit-learn standards.
+See [1] for complete set of conventions.
+[1] http://scikit-learn.org/stable/developers/
+"""
+
+
+from .ldamodel import LdaTransformer  # noqa: F401
```

Comment:
A line having `noqa` doesn't issue flake8 errors. Writing `noqa: F401` saves us from flake8 errors saying that we have imports (e.g. `LdaTransformer`) which we haven't used anywhere in the file.